host/onebox.go

```diff
@@ -707,7 +706,23 @@ func (c *cadenceImpl) startWorker(hosts map[string][]string, startWG *sync.WaitG
 	}
 	params.PublicClient = workflowservice.NewWorkflowServiceClient(connection)
 
-	service := service.NewOneboxService(params)
+	service, err := resource.New(
```

Comment:
This is little bit hacky. I copied this code from `worker/service.go` because I can't call `Start` on it due to customer workers start logic bellow. Let it be like this for now (onebox is for tests only) but eventually entire onebox concept should be changed.